catalog/ui/search-ui/standard/src/main/webapp/css/style.css

```diff
@@ -164,16 +164,16 @@ div.modal-body div.preferenceTabButtons {
     standardized selectors and properties and non-standard and prefixed selectors
     and properties."
 */
```

Comment:
I would move all these input range stylings into a new .less file under ```ddf/catalog/ui/search-ui/standard/src/main/webapp/less/components/```.  You could call it ```range-input.less```.